### ISSUE TITLE
GitHub Demo

### ISSUE THREAD

**Issue by froi** [MERLTech/github-demo#1]
Hello everybody 👋🏼 

As part of our demo we are going to be using this GitHub issue for agenda! 🎉 

## Agenda

- [x] What are Issues and how to use them?
- [x] What are Pull Requests and how to use them?
- [x] What does it mean to merge a branch?
- [x] Licenses in layman's terms from a layman (not a lawyer)

### GitHub Guides

- [Hello World](https://guides.github.com/activities/hello-world/)
- [Mastering Issues](https://guides.github.com/features/issues/)
- [Mastering Markdown](https://guides.github.com/features/mastering-markdown/)
- [Mastering the GitHub Flow](https://guides.github.com/introduction/flow/)

You can find all of these at [guides.github.com](https://guides.github.com/)

### Licenses

- [Choose A License](https://choosealicense.com/)
- [Creative Commons - Share your work](https://creativecommons.org/share-your-work/)

**Comment by froi** [comment]
I like this

**Comment by froi** [comment]
@malakumar85 what do you think about this???

**Comment by mmahomar** [comment]
Can comments be deleted or updated?
